.env.example

```
@@ -1,7 +1,10 @@
-DATABASE_URL='postgresql://<user>:<pass>@<db-host>:<db-port>'
+DATABASE_URL='postgresql://<user>:<pass>@<db-host>:<db-port>/<db-name>'
```

```suggestion
DATABASE_URL='postgresql://<user>:<pass>@<db-host>:<db-port>'
```
This seems out of the scope of this PR